model/src/main/java/org/projectnessie/model/Content.java

```diff
@@ -40,7 +41,7 @@
     discriminatorProperty = "type")
 @JsonSubTypes({@Type(IcebergTable.class), @Type(DeltaLakeTable.class), @Type(IcebergView.class)})
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
-public abstract class Content {
+public abstract class Content implements Serializable {
```

Comment:
I wouldn't mind to make all model objects serializable - but please in a separate PR.